### ISSUE TITLE
Can I drag&drop file in mobox like Windows?

### ISSUE THREAD

**Issue by Imsunnn** [olegos2/mobox#51]
Please, I don't know how to drag&drop, please tell me if possible

**Comment by GrigoryNot2** [comment]
> Please, I don't know how to drag&drop, please tell me if possible

Нет. Это невозможно. Никто не будет терять время и ресурсы для реализации этой бесполезной функции. Кроме вас не кому.
